app/ui/src/main/java/com/fsck/k9/account/KoinModule.kt

```diff
@@ -0,0 +1,8 @@
+package com.fsck.k9.account
+
+import org.koin.dsl.module.applicationContext
+
+val accountModule = applicationContext {
+    factory { AccountRemover(get(), get(), get()) }
```

Comment:
Why is this a `factory` instead of a `bean`? Do you plan to introduce an account scope or something like that?

Reply:
The idea was that we don't need an instance of this hanging around most of the time.